### ISSUE TITLE
Project Listing not showing our pictures

### ISSUE THREAD

**Issue by 0WN463** [nusCS2113-AY1819S1/forum#45]
The link to my image on https://nuscs2113-ay1819s1.github.io/website/admin/projectList.html
is 
https://cs2113-ay1819s1-f10-1.github.io/main/images/0wn463.png
However, on our repo, it is 
https://cs2113-ay1819s1-f10-1.github.io/main/images/0WN463.png
As you can see, the difference is the case of the letters. I wonder if it is a error on our side or on your side? Since my user is clearly in uppercase and I vaguely remember submitting my username as uppercase during the pre-module survey thingy.
Also, since the bot is able to correctly identify me for the PR for LO's and feedback report on the organization's progress, I don't suppose I specify it in lower case.
So bottom line is, should the image be in completely lowercase or is it a fault on your side?

**Comment by QzSG** [comment]
Why so passive aggressive?

> About Us page:
This page is used for module admin purposes. Please follow the format closely to avoid penalties.
Replace info of SE-EDU developers with info of your team, including a suitable photo as described here.
Including the name/photo of the supervisor/lecturer is optional.
The photo of a team member should be doc/images/githbub_username_in_lower_case.png e.g. docs/images/akshay.png.

So whose fault is it? 🤔

**Comment by 0WN463** [comment]
Well, didn't mean to come off that way. It's either I'm friggin blind or that's not prominent enough. 
Then again, I just read from the other issue about ProjectList
>Have you named your own photos with your github username (e.g: my github user name: okkhoy. Hence, the photo should be okkhoy.png)

It wasn't specified there and I guess I got confused by that
